docs/user-guide/faq.md

```diff
@@ -34,6 +34,10 @@ There are example of using the PostCSS plugin within the [docs](/docs/user-guide
 
 stylelint can *parse* any the following non-standard syntaxes: SCSS, Less and SugarSS. Refer to the [docs](/docs/user-guide/css-processors.md#parsing-non-standard-syntax) on how to configure stylelint to parse one of these syntaxes.
 
+## How do I lint styles within `style` tags?
+
+stylelint can *extract and lint* any styles within `<style>` tags. Refer to the [docs](/docs/user-guide/node-api.md#extractStyleTagsFromHtml) for turning on this feature. You can also use the cli's `--extract` option to turn this feature on.
```

Comment:
How about for the second sentence: "This can be done via the Node API and the CLI." (with links)